docs/guides/oracle-operator-manual.md

```diff
@@ -6,14 +6,13 @@ The daemon also fetches historical stETH token price (shifted by fifteen blocks)
 
 ## TL;DR
 
-1. Generate an Ethereum address and propose it as an oracle address via the "Add Member" button in the app UI: [Mainnet] / [Görli].
+1. Generate an Ethereum address.
 2. Facilitate the DAO members to approve your oracle address.
```

Comment:
Ask DAO members to prepare a vote to add addresses to the oracle set.

And I guess they have to share this address somewhere first. In case of mainnet it should be twitter and the research forum, in case of testnet it may be telegram or discord

Reply:
Fixed

Reply:
fixed